experimental/harfbuzz/src/lib.rs

```diff
@@ -166,7 +167,9 @@ unsafe extern "C" fn icu4x_hb_unicode_script(
     let script_data: &ScriptDataForHarfBuzz = &*(user_data as *mut ScriptDataForHarfBuzz);
     let script: Script = script_data.script_map.as_borrowed().get32(unicode);
     let enum_to_name_mapper = script_data.enum_to_name_mapper.as_borrowed();
-    let name: &str = enum_to_name_mapper.get(script).unwrap_or("Zzzz");
+    let name: TinyStr4 = enum_to_name_mapper
+        .get(script)
+        .unwrap_or(tinystr!(4, "Zzzz"));
     hb_script_from_string(
```

Comment:
Note: Henri wanted this to cast the `TinyStr4` directly to a `hb_script_t` and avoid the call into `hb_script_from_string` but I think that should probably be its own PR with more tests.